frontend/test/metabase/scenarios/question/nested.cy.spec.js

```diff
@@ -255,7 +257,7 @@ describe("scenarios > question > nested", () => {
 
     const baseQuestionDetails = {
       name: "14787",
-      query: ordersJoinProductsQuery,
+      query: { ...ordersJoinProductsQuery, limit: 5 },
```

Comment:
Note:
This line had nothing to do with the fix. The test was just insanely wasteful for no apparent reason. Sorry for the potential confusion.